### PR TITLE
support for Azure commands : Create / Delete Resource group

### PR DESCRIPTION
**Pull request by dikhakha**
Added support for Azure commands : Create / Delete Resource group

### CODE REVIEW

File: arm/entrypoint.sh [hunk 1]
```diff
@@ -2,60 +2,77 @@
...
-# Create Resource group if it does not exists
-
-RESOURCE_GROUP_EXISTS=$(az group exists -n "$AZURE_RESOURCE_GROUP")
-if [[ $RESOURCE_GROUP_EXISTS == "false" ]]
+if [[ -z "$AZURE_RESOURCE_GROUP" ]]
 then
-  if [[ -z "$RESOURCE_GROUP_LOCATION" ]]
+  echo "AZURE_RESOURCE_GROUP is not set." >&2
+  exit 1
+else
```
Comment (sachinma) [inline arm/entrypoint.sh]: Do you need this else. Meaning since you are exiting from if above, you can avoid the nesting

File: arm/entrypoint.sh [hunk 2]
```diff
@@ -2,60 +2,77 @@
...
+  if [[ -z "$AZURE_RG_COMMAND" ]] || [[ ${AZURE_RG_COMMAND,,} == 'create' ]]
```
Comment (sachinma) [inline arm/entrypoint.sh]: I think Azure_RG_command is mandatory without any default

Comment (sachinma) [inline arm/entrypoint.sh]: what does ,, indicate?

File: arm/entrypoint.sh [hunk 3]
```diff
@@ -2,60 +2,77 @@
...
+      else
+        az group deployment create -g "$AZURE_RESOURCE_GROUP" --name "$DEPLOYMENT_NAME" --template-file "$TEMPLATE_FILE" --parameters "$PARAMETERS"
+      fi
+    fi
+  elif [[ ${AZURE_RG_COMMAND,,} == 'delete' ]]
   then
-    echo "Template file ${TEMPLATE_FILE} does not exists." >&2
-    exit 1
+    echo "Executing commands to Delete resource group."
+    az group delete -n "$AZURE_RESOURCE_GROUP" --no-wait --yes
```
Comment (sachinma) [inline arm/entrypoint.sh]: do we need --no-wait experience. I believe, we should wait

**Review by sachinma — CHANGES_REQUESTED** [review]

File: arm/entrypoint.sh (hunk 3 above)
Comment (dikhakha) [reply arm/entrypoint.sh]: Spec question here. Do we want to wait for long running clean up operation ?

File: arm/entrypoint.sh (hunk 2 above)
Comment (dikhakha) [reply arm/entrypoint.sh]: that evaluates the AZURE_RG_COMMAND to lower case. i added this for case insensitive command

File: arm/entrypoint.sh (hunk 1 above)
Comment (dikhakha) [reply arm/entrypoint.sh]: yes this can be avoided, will make the change

File: arm/entrypoint.sh (hunk 3 above)
Comment (dikhakha) [reply arm/entrypoint.sh]: waiting for delete operation now.

**Review by sachinma — APPROVED** [review]

**Pull request merged**